Cue/app/tabs/search/SearchListViewItem.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+'use strict';
+
+import React from 'react'
+import { Image, View, Text, TouchableOpacity, Navigator, Platform, TouchableNativeFeedback, TouchableHighlight } from 'react-native'
+
+import type { DeckMetadata } from '../../api/types'
+
+import CueColors from '../../common/CueColors'
+import CueIcons from '../../common/CueIcons'
+
+import DeckThumbnail from '../../common/DeckThumbnail'
+import DeckRating from '../../common/DeckRating'
+
+const styles = {
+  itemContainer: {
+    borderBottomWidth: 2,
+    borderColor: '#eee',
+    marginTop: 30,
+    flexDirection: 'row',
+    flex: 1,
+  },
+  textContainer: {
+    flex: 1,
+    flexDirection: 'column',
+    marginBottom: 30
+  },
+  firstText: {
+    fontSize: 16,
+    flex: 1,
+  },
+  cardContainer: {
+    marginHorizontal: 16,
+    marginBottom: 30,
+  }
+}
+
+export default class SearchListViewItem extends React.Component {
+  props: {
+    navigator: Navigator,
+    deck: DeckMetadata
+  }
+
+  render() {

```

Comment:
For some reason, the list view items don't want to render completely on iOS. I can't troubleshoot right now, but I'm ok with just creating a separate issue to fix it (and also the search box).

<img width="391" alt="screen shot 2017-03-15 at 10 57 06 pm" src="https://cloud.githubusercontent.com/assets/13400887/23979894/bdf51d74-09d2-11e7-8cea-f1f32d67893f.png">